src/sourceSystem.h

```diff
@@ -5,6 +5,7 @@
 #include "qssintegrator.h"
 #include "quasi2d.h"
 #include "callback.h"
+#include "grid.h"
```

Comment:
I don't think this extra include is needed, right?

Reply:
Yes, forgot to remove when moving strain terms to convectionSystem.